docs/data-sources/synthetic_monitoring_probes.md

```diff
@@ -14,13 +14,18 @@ Data source for retrieving all probes.
 
 ```terraform
 data "grafana_synthetic_monitoring_probes" "main" {}
+
+data "grafana_synthetic_monitoring_probes" "not_deprecated" {
+  filter_deprecated = true
+}
 ```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Optional
 
+- **filter_deprecated** (Boolean) If true, only probes that are not deprecated will be returned. Defaults to `false`.
```

Comment:
I would lean to keeping it to `true` by default. why? ideally we should not default to scheduling checks to deprecated probes.
```suggestion
- **filter_deprecated** (Boolean) If true, only probes that are not deprecated will be returned. Defaults to `true`.
```

Reply:
deprecated probes will eventually be removed and turned off.

Reply:
Yeah, I was worried about unexpected changes to users of the provider but on the other hand, when they are removed, users will have changes in any case.

👍 will default to true